x-pack/plugins/security_solution/public/management/pages/trusted_apps/view/components/create_trusted_app_form.test.tsx

```diff
@@ -252,55 +247,55 @@ describe('When using the Trusted App Form', () => {
   });
 
   describe('the Policy Selection area', () => {
-    it('should show loader when setting `policies.isLoading` to true', () => {
+    beforeEach(() => {
+      const policy = generator.generatePolicyPackagePolicy();
+      policy.name = 'test policy A';
+      policy.id = '123';
+
+      formProps.policies.options = [policy];
+    });
+
+    it('should have `global` switch on if effective scope is global and policy options hidden', () => {
+      render();
+      const getGlobalButton = (dataTestSub: string = dataTestSubjForForm): HTMLButtonElement => {
+        return renderResult.getByTestId(
+          `${dataTestSub}-effectedPolicies-global`
+        ) as HTMLButtonElement;
+      };
```

Comment:
(optional) These functions (this one and the one below in the next test to `getPerPolicyButton`) seem unnecessary. You can just do the selection of the element and store it locally as in: 

```suggestion
      const globalButton = renderResult.getByTestId(`${dataTestSub}-effectedPolicies-global`) as HTMLButtonElement;
```